File: benchmark/corpus/12_indirect_authority/d_narrows/1.0.0/report.py
# report 1.0.0: formats a total, with a label read from the data directory.


def label():
    try:
        with open("{granted}/notes.txt") as f:
            return f.read().strip()
    except OSError:
        return "total"


def total_line(total):
    return f"{label()}: {total}"

File: benchmark/corpus/12_indirect_authority/d_narrows/1.0.0/report.js
// report 1.0.0: formats a total, with a label read from the data directory.
function label() {
  try {
    return Deno.readTextFileSync("{granted}/notes.txt").trim();
  } catch (_e) {
    return "total";
  }
}

export function totalLine(total) {
  return `${label()}: ${total}`;
}
